pkg/trace/gcp.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2020 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package trace
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"go.opentelemetry.io/otel/api/trace"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"k8s.io/klog"
+
+	texporter "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace"
+	"github.com/pkg/errors"
+	"go.opentelemetry.io/otel/api/global"
+)
+
+const (
+	projectEnvVar  = "MINIKUBE_GCP_PROJECT_ID"
+	parentSpanName = "minikube start"
```

Comment:
what is this ? do u mind adding comments ?
would it work if minikube's binary is different and user call is 

`minikube-darwin-amd64 start`

 /

Reply:
So this is just a name for the span as it appears in Cloud Trace to provide info around what was being traced. It won't have any effect on running minikube, added a comment for clarity.